Foundation/NSHTTPCookie.swift

```diff
@@ -370,7 +397,9 @@ public class NSHTTPCookie : NSObject {
         @result The comment of the receiver, or nil if the receiver has no
         comment.
     */
-    public var comment: String? { NSUnimplemented() }
+    public var comment: String? {
+        return (_properties[NSHTTPCookieComment] as! NSString)._swiftObject)
```

Comment:
Trying to access these such optional values would throw a runtime exception with the `as!` statement